commands/operator-sdk/cmd/new.go

```diff
@@ -179,6 +181,23 @@ func doScaffold() {
 		log.Fatalf("failed to create %v: %v", dockerfilePath, err)
 	}
 
+	// generate pkg/build/build.sh
+	buildScriptPath := filepath.Join(buildDir, "build.sh")
+	buildScriptgen := scaffold.NewbuildCodegen(
```

Comment:
Minor nit: Just noticed you might want to fix the camel case here and above in other places.
`buildScriptGen`

Reply:
yeah, good catch!